osquery/tables/applications/browser_utils.cpp

```diff
@@ -74,9 +78,34 @@ void genExtension(const std::string& uid,
     }
   }
 
+  // Fetch the permission array from the manifest file
+  std::string permission_list;
```

Comment:
You could collect strings with separator without string stream. Something like [ref](https://stackoverflow.com/a/1833499). If you like to :)

Or just with string.append:
```c++
std::string permission_list;
for (...) {
  permission_list.append(perm_obj.second.get_value<std::string>());
  permission_list.append(", ");
}
permission_list.pop();
permission_list.pop();
```

Reply:
Hey @alessandrogario !
You could put all permissions to the vector and join the results after.
```c++
std::vector<std::string> permission_list;
for (...) {
  permission_list.push_back(
    perm_obj.second.get_value<std::string>()
  );
}
...
r[kExtensionPermissionKey] = boost::algorithm::join(permission_list, ", ");
```
Yes, I would prefer to use `string::append` instead of `stringstream`, because it is notably faster.
Up to you ;)

Reply:
Are we all good here @akindyakov , should I merge?